### ISSUE TITLE
About the entropy minimization

### ISSUE THREAD

**Issue by Solacex** [valeoai/ADVENT#1]
Hello~
I notice that the log you are using in the entropy minimization is log2() rather than log(), have you tried torch.log() ?

**Comment by tuanhungvu** [comment]
Hello @Solacex , we haven't tried other bases. Using another base will simply scale the gradients by a constant scalar, which is no different from scaling the learning rate of entropy loss.
T-H

**Comment by Solacex** [comment]
> Hello @Solacex , we haven't tried other bases. Using another base will simply scale the gradients by a constant scalar, which is no difference from scaling the learning rate of entropy loss.
> T-H

Thank you for your prompt reply~

**Comment by soans1994** [comment]
@tuanhungvu @Solacex 

In the paper there are 2 methods, direct entropy minimization and adversarial training. Where is the first direct entropy minimization training code?
There is only adversarial entropy minimization here?

thank you
